examples/todomvc/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+cargo +nightly build --target wasm32-unknown-unknown --verbose --release &&
+wasm-bindgen ../../target/wasm32-unknown-unknown/release/todomvc.wasm --out-dir . &&
+npm install &&
+npm run serve
```

Comment:
Could this copy the `build.sh` scripts from the other examples to have a few extra comments as well? (also use `wams-bindgen` as a path dependency)